pallets/tips/src/tests.rs

```diff
@@ -157,6 +159,36 @@ impl pallet_treasury::Config for Test {
     type MaxApprovals = MaxApprovals;
     type ProposalBondMaximum = ();
 }
+
+const MILLICENTS: Balance = 10_000_000_000_000;
+const CENTS: Balance = 1_000 * MILLICENTS;
+const DOLLARS: Balance = 100 * CENTS;
+
+parameter_types! {
+    pub const ClassDeposit: Balance = 100 * DOLLARS;
+    pub const InstanceDeposit: Balance = 1 * DOLLARS;
+    pub const KeyLimit: u32 = 32;
+    pub const ValueLimit: u32 = 256;
+    pub const StringLimit: u32 = 50;
+}
+
+impl pallet_uniques::Config for Test {
+    type Event = Event;
+    type ClassId = u32;
+    type InstanceId = u32;
+    type Currency = Balances;
+    type ForceOrigin = frame_system::EnsureRoot<u128>;
+    type ClassDeposit = ConstU64<2>;
```

Comment:
These 4 `deposit` values are of type `BalanceOf`, maybe you should set a `1*DPR` or `2*DPR`, just write a 1 or 2 is a very small value.

Reply:
the test config is small value, but big value in runtime/src/lib.rs
```
    pub const ClassDeposit: Balance = 100 * DOLLARS;
    pub const InstanceDeposit: Balance = 1 * DOLLARS;
    
    pub const MetadataDepositBase: Balance = 10 * DPR;
    pub const MetadataDepositPerByte: Balance = 1 * DPR;
```